Objects/longobject.c

```diff
@@ -239,6 +229,39 @@ PyLong_FromLong(long ival)
     return (PyObject *)v;
 }
 
+/* Convert result of add/subtract of medium values
+ * to a PyLong.
+ */
+static inline PyObject *
+PyLong_FromDigitPlusOneBit(sdigit x)
```

Comment:
I don't understand the function name. What does it mean "a digit plus 1 bit"? sdigit type contains a single digit, no?

Other function names are PyLong_From<C type name>. Maybe rename it to _PyLong_FromSDigit().